package.json

```diff
@@ -46,7 +46,8 @@
     "gen_env": "node ./build/commands/scripts/genEnv.js",
     "gen_gradle": "node ./build/commands/scripts/commands.js gen_gradle",
     "ios_pack_js": "webpack --config ios/brave-ios/webpack.config.js",
-    "ios_bootstrap": "node ./build/commands/scripts/iosCommands.js ios_bootstrap"
+    "ios_bootstrap": "node ./build/commands/scripts/iosCommands.js ios_bootstrap",
+    "build_policy_templates": "node ./build/commands/scripts/commands.js build --target=brave_group_policy_templates"
```

Comment:
1. please avoid adding "shortcut" commands for cases like this. You can just call `npm run build -- --target=brave_group_policy_templates` where you need to. It's fine to have a separate `npm run` command if we add some custom logic in `build/commands/**/*.js`, but here we're just calling a GN target.

2. consider renaming the GN target to have a verb, for ex.: `generate_brave_group_policy_templates`.

Reply:
I don't have a strong opinion on 2. but [concur](https://github.com/brave/brave-core/pull/25699#discussion_r1773575641) with 1.

Reply:
Removed! Thanks for the feedback - @mherrmann had raised. I went ahead and removed. We can continue to use `npm run build -- --target=brave/components/policy:pack_policy_templates`